plugins/toolkit/jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -240,4 +264,29 @@ class JavaLocalLambdaRunConfigurationIntegrationTest(private val runtime: Lambda
         expectedOutput = input.uppercase(),
         addBreakpoint = { projectRule.addBreakpoint() }
     )
+
+    private fun createMinimalGradleBuildSetup(testProjectDir: File) {
+        val buildGradleFile = File(testProjectDir, "build.gradle.kts")
+        buildGradleFile.writeText(
+        """
+            plugins {
+                java
+            }
+ 
+            repositories {
+                mavenCentral()
+            }
+        """.trimIndent()
+        )
+    }
+
+    private fun createGradlePropertiesFile(gradleUserHome: File) {
+        val gradlePropertiesFile = File(gradleUserHome, "gradle.properties")
+        gradlePropertiesFile.writeText(
+            """
+            org.gradle.jvmargs=-Xmx4096m
+            gradle.user.home=${gradleUserHome.absolutePath}
```

Comment:
something that is also unclear at the moment is if we are actually allowed to modify the user home in this manner, or if this is too late because gradle has already selected a user home

Reply:
So, in this case shall we try using Env Variable GRADLE_USER_HOME, so it can be set before Gradle initializes. 
will test this out